lms/envs/common.py

```diff
@@ -3432,6 +3432,16 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 COURSE_ABOUT_VISIBILITY_PERMISSION = 'see_exists'
 
 DEFAULT_COURSE_VISIBILITY_IN_CATALOG = "both"
+
+# .. toggle_name: DEFAULT_MOBILE_AVAILABLE
+# .. toggle_implementation: DjangoSetting
+# .. toggle_default: False
+# .. toggle_description: This specifies if the courses are available for mobile by default. To make any individual
+#  course available for mobile one can set the value of Mobile Course Available to true in Advanced Settings from the
```

Comment:
I think we might need at least one more space in the indent fir this to work correctly.

Reply:
I have added one more space but[ how-to for document toggles](https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html) suggests that the `toggle_description` requires two empty spaces. So, shall we change the documentation with three spaces? Correct me if I am wrong anywhere.

Reply:
@jramnai: Thanks for the update.  Ideally we can fix the parsing so that this would have worked as-is.  If not, we'll update the how-to. The reason I thought it wouldn't work was because we need to make what I thought was a similar fix here: https://github.com/edx/edx-platform/pull/26172/files#r565010977
FYI: @regisb

Reply:
I didn't know how to phrase the requirement better: "every line after the first is prefixed by at least two empty spaces, **plus the base indentation of the first line**"
I guess we could add "(thus usually at least three in total)".

Reply:
I'll add a note to self to see if we can add a little clarity to the docs.  Thanks for clearing this up @regisb.